db/migrate/20200529144649_add_latest_invoice_id_to_stripe_subscription.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddLatestInvoiceIdToStripeSubscription < ActiveRecord::Migration[6.0]
+  def change
+    add_column :stripe_subscriptions, :latest_stripe_invoice_id, :string
+    add_index :stripe_subscriptions, :latest_stripe_invoice_id
```

Comment:
Couldn't this be simplified with `add_reference`?
```suggestion
    add_reference :stripe_subscriptions, :latest_stripe_invoice, type: :string
```